app/controllers/application_controller.rb

```diff
@@ -304,8 +304,10 @@ def set_sentry_context
         is_bot: user_agent.bot?,
         request_id: request.request_id,
         user_id: current_user&.id,
-        client_id: current_client&.id
+        client_id: (current_client&.id || current_intake&.id)
```

Comment:
I'm guess we can just revert this back to what it was since we aren't trying to use current_client in the state file intercom partial anymore